main.go

```diff
@@ -365,7 +394,27 @@ func runCLIApp(c *cli.Context) (err error) {
 			err = fmt.Errorf("daemonize.Run: %w", err)
 			return
 		} else {
-			logger.Infof(SuccessfulMountMessage)
+			if isDynamicMount {
```

Comment:
This code is a bit complex. Should we use an interface with foreground and background handling as two different implementations?

Reply:
It'll be an overkill I think given the code size.